scripts/generate-wiki-markdown.js

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+const https = require('https')
+
+const repoPath = 'nuxt/nuxt.js'
+const pkgNamespace = '@nuxt'
+const branches = [
+  {
+    name: '2.x',
+    title: 'Stable'
+  },
+  {
+    name: '',
+    title: 'Edge',
+    pkgSuffix: '-edge'
+  }
+]
+
+const template = ({ tables, vars, extra }) => `
+<!-- /* this page is automatically generate */ -->
+
+## Distributions
+${tables.dist}
+
+## Core packages
+${tables.pkgs}
+
+${extra}
+${vars}
+`
+
+const options = {
+  '-d': 'dependencies',
+  '-p': 'packageSize'
+}
+
+const optionKey = process.argv.find(arg => !!options[arg])
+
+if (!optionKey) {
+  // eslint-disable-next-line no-console
+  console.log(`Usage: ${path.basename(__filename)} -d | -p
+  -d    Generate markdown for dependencies page
+  -p    Generate markdown for package sizes page
+`)
+  process.exit(1)
+}
+
+const generateOption = options[optionKey]
+
+function arrayToTable (tableArray) {
+  let table = ''
+  tableArray.forEach((row, index) => {
+    table += row.join('|')
+    table += '\n'
+
+    if (!index) {
+      // header border
+      table += row.map(c => '-').join('|')
+      table += '\n'
+    }
+  })
+
+  return table
+}
+
+function request (url) {
```

Comment:
Why not using axios or node-fetch?

Reply:
Cause I prefered 0 dependencies so this script can easily be used for eg nuxt/press or non-nuxt monorepo's as well

Reply:
We could integrate this script maybe in the jsless package.js package if thats being extracted, then it would be ok probably

Reply:
Yep! I really want to externalize our maintenance at a point and your wiki generator is AWESOME for that!